issue found on: temp/analyser.go:188:1: paramTypeCombine

an issue has been detected in source code: ./temp/analyser.go:188:1: paramTypeCombine: func(entries []AggregatorLogEntry, notRead []AggregatorLogEntry) could be replaced with func(entries, notRead []AggregatorLogEntry)

--- a/temp/analyser.go
+++ b/temp/analyser.go
@@ -184,7 +184,7 @@ func printConsumedEntries(entries, notRead []AggregatorLogEntry) {
 	}
 }
 
-func printReadEntries(entries []AggregatorLogEntry, notRead []AggregatorLogEntry) {
+func printReadEntries(entries, notRead []AggregatorLogEntry) {
 	for _, entry := range notRead {
 		printReadEntry(entry)
 		printErrorsForMessageWithOffset(entries, entry.Offset)

